docs-v2/reference/api-configuration.mdx

```diff
@@ -193,7 +193,7 @@ slack:
 </ResponseField>
 
 <ResponseField name="auth_mode" type="string" >
-    The authentication mode. Must be one of: "API_KEY", "APP", "APP_STORE", "BASIC", "NONE", "OAUTH1", "OAUTH2", "OAUTH2_CC", "CUSTOM", "TBA", "TABLEAU", "JWT", "BILL".
+    The authentication mode. Must be one of: "API_KEY", "APP", "APP_STORE", "BASIC", "NONE", "OAUTH1", "OAUTH2", "OAUTH2_CC", "CUSTOM", "TBA", "TABLEAU", "JWT", "BILL", "TWOSTEP".
```

Comment:
are we set on the name `TWOSTEP`? @khaliqgant @bodinsamuel @hassan254-prog It is harder to change once used. if yes should it be `TWO_STEP`?

Reply:
Since token-based authentication is a security process involving two steps; authentication and token generation, 
I suggest naming it `TWO_STEP`. Perhaps Khaliq and Samuel have other reservations?

Reply:
`TWO_STEP` is good for a name for me.